lib/src/chart/line_chart/line_chart_data.dart

```diff
@@ -956,6 +959,7 @@ class LineTouchData extends FlTouchData {
     this.touchSpotThreshold = 10,
     this.fullHeightTouchLine = false,
     this.handleBuiltInTouches = true,
+    this.stickyTouchIndicator = false,
```

Comment:
I don't understand what it is doing when it is `false`?
It should be like previous? or `false` means working in sticky mode?

Reply:
false is the default value (not sticky -- no breaking changes) but if you set it to true like I did in sample 8, it becomes sticky.

Reply:
sorry the conditional need `!` in it. just updated.

Reply:
Yes, I guessed you may forgot the `!`, that's why I'm asked.
Thanks!